cloudygames/views.py

```diff
@@ -44,31 +46,35 @@ def get_queryset(self):
                 user = self.request.user
                 owned_games_id = GameOwnership.objects.filter(
                     user=user).values_list('game__id', flat=True)
-                return Game.objects.filter(pk__in=owned_games_id)
+                return qs.filter(pk__in=owned_games_id)
```

Comment:
can you order this by name as well so it will be consistent with the all games one